### PR TITLE
destination-dev-null: add to OSS registry

### PR DESCRIPTION
**Pull request by stephane-airbyte**
## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
* **#45715** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/45715?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stephane-airbyte and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/integrations/destinations/dev-null.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ The OSS and Cloud variants have the same version number starting from version `0
 
 | Version | Date       | Pull Request                                             | Subject                                                   |
 | :------ | :--------- | :------------------------------------------------------- | :-------------------------------------------------------- |
+| 0.6.1   | 2024-09-20 | [45715](https://github.com/airbytehq/airbyte/pull/45715) | add destination to cloud registry |
```
Comment (johnny-schmidt) [inline docs/integrations/destinations/dev-null.md:52]:
```suggestion
| 0.6.1   | 2024-09-20 | [45715](https://github.com/airbytehq/airbyte/pull/45715) | add destination to loss registry |
```

**Review by johnny-schmidt — APPROVED** [review]
approved with a nit about the changelog

**Comment by stephane-airbyte** [comment]
### Merge activity

* **Sep 20, 7:54 PM EDT**: @stephane-airbyte merged this pull request with [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/45715).

**Pull request merged**